chroma-manager/Makefile

```diff
@@ -208,7 +208,12 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
-requirements: requirements.txt
+install_requirements: requirements.txt
```

Comment:
that is a valid concern, although the disadvantages of updating those critical packages are probably minimal